lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py

```diff
@@ -15,7 +15,7 @@
 
 DOCUMENTATION = '''
 ---
-module: azure_rm_virtualmachine_extension
+module: azure_rm_virtualmachineextension
```

Comment:
you mentioned this one was added before ansible 2.8. so any alias to link old name to guarantee backward  compatibility?

Reply:
I will check that one...

Reply:
perhaps i should ask @felixfontein to take a look if there's anything else that needs to be done here.

Reply:
added note with link in the documentation

Reply:
Sorry for replying so late. You've already added a note, that's IMO always a good idea. Two things you could also do:
 1. add a deprecation warning which is emitted whent the old module name is used, and which tells the user the new name to use;
 2. add an entry into the porting guide.

Both don't seem to be mandatory; I've always used 1. before, but never 2. for renames, though @gundalow recently recommended 2. in some discussion. For 1., see https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/crypto/acme/acme_account_info.py#L130-L131 for how this can be done.

By having the deprecation link, an alias will already be added to the docs page; see [here](https://docs.ansible.com/ansible/devel/modules/acme_account_info_module.html#synopsis). This is added because of the symlink.

Reply:
thanks a lot @felixfontein !